tests/dedicatedvstests/l7_dedicated_crd_test.go

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2019-2020 VMware, Inc.
```

Comment:
nit: please update the year for the copyright. This can be taken up in a subsequent PR too.